src/components/ImmatriculationHistory.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+    <div v-if="!busy">
+        <div v-if="chronologicalList.length > 0">
+            <div v-for="(pair, index) in chronologicalList" :key="pair">
+                <immatriculation-history-entry
+                    class="w-1/2"
+                    :is-first-row="index == 0"
+                    :is-last-row="index == chronologicalList.length - 1"
+                    :fields-of-study="pair.fieldsOfStudy"
+                    :semester="pair.semester"
+                />
+            </div>
+        </div>
+        <div v-else>
+            <label class="text-lg">There is no matriculation data, yet!</label>
+        </div>
+    </div>
```

Comment:
```suggestion
    <div v-if="busy">
        <loading-component />
    </div>
    <div v-else">
        <div v-if="chronologicalList.length > 0">
            <div v-for="(pair, index) in chronologicalList" :key="pair">
                <immatriculation-history-entry
                    class="w-1/2"
                    :is-first-row="index == 0"
                    :is-last-row="index == chronologicalList.length - 1"
                    :fields-of-study="pair.fieldsOfStudy"
                    :semester="pair.semester"
                />
            </div>
        </div>
        <div v-else>
            <label class="text-lg">There is no matriculation data, yet!</label>
        </div>
    </div>
```

Reply:
this is intended, as the busy boolean is emitted to the parent component